src/main/webapp/course.html

```diff
@@ -85,7 +85,12 @@ <h1 class="p-4 text-center">Add Your Review</h1>
                 class="form-control mb-5"
                 rows="5"
                 placeholder="Write Opinion Here"
+                required
               ></textarea>
+              <div class="valid-tooltip">Looks good!</div>
```

Comment:
I don't see anywhere selectively hiding/showing these tooltips, where is this handled?

Reply:
It was supposed to automatically take care of it adding the class="needs-validate" to the form and adding the bootstrap source to the HTML. I have encountered problem with getting this to display tho so I will work on this on a separate PR to get this to work.